app/events/[event_id]/styles.ts

```diff
@@ -0,0 +1,124 @@
+'use client';
+
+import NextImage from 'next/image';
+import styled from 'styled-components';
+import COLORS from '@/styles/colors';
+import { H4, H5, P, SMALL } from '@/styles/text';
+
+export const BackImage = styled(NextImage)`
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 1.5rem;
+  height: 1.5rem;
+  margin: 1rem;
+`;
+
+export const ImageWrapper = styled.div`
+  position: relative;
+  width: 100%;
+  height: auto;
+  display: flex;
+`;
+
+export const EventImage = styled(NextImage)`
+  width: 100%;
+  height: auto;
+  max-height: 11.5rem;
+  object-fit: cover;
+`;
+
+export const GradientOverlay = styled.div`
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background: linear-gradient(
+    0deg,
+    rgba(0, 0, 0, 0.35) 0%,
+    rgba(0, 0, 0, 0.35) 100%
+  );
+`;
+
+export const Page = styled.div`
+  background-color: ${COLORS.gray1};
+  flex-direction: column;
+  min-width: 100%;
+  min-height: 100svh;
+  overflow: hidden;
+  margin-bottom: 3.75rem;
+`;
+
+export const Container = styled.div`
```

Comment:
- `border-radius` field helps us round the corners!
- deleting `margin-bottom` field in `Page` since bottom padding is directly being added to this container
- negative margins help us overlap elements and `z-index` sets the order of overlapped elements. the default z-index value is 0 and higher z-index values bring elements to the front, while lower values push them to the back
- i'm making the background white for now so that the container doesn't blend in with the default background color
```
export const Container = styled.div`
  z-index: 10;
  margin: -1.5rem 0;
  position: relative;
  padding: 1.5rem 2rem 4rem 2rem;
  display: flex;
  flex-direction: column;
  border-radius: 20px 20px 0 0;
  background: white;
`;
```